LoopKit/InsulinKit/CachedInsulinDeliveryObject+CoreDataClass.swift

```diff
@@ -61,6 +61,30 @@ class CachedInsulinDeliveryObject: NSManagedObject {
         }
     }
 
+    var scheduledTempBasalRate: HKQuantity? {
```

Comment:
Does 'programmedTempBasalRate' convey the meaning more clearly? There's almost always a difference between the programmed/scheduled rate and delivered units / time because of the pulsed nature of delivery.  HealthKit tracks the units, and a duration. Tracking the programmed temp rate lets us recover what was originally programmed.  If you think the current name is confusing, and `programmedTempBasalRate` has more obvious semantics, I'll change it. It means a data model change (migration, maybe a lightweight one) and the HKMetadata key change, but that should all be fine.

Reply:
Yes, "programmedTempBasalRate" is perfect, but if a migration is necessary, then don't bother. A comment above the property is just as good. :)

Reply:
:+1: